apps/oxlint/src/snapshots/--print-config@oxlint.snap

```diff
@@ -0,0 +1,142 @@
+    "no-cond-assign": "warn",
+    "no-useless-catch": "warn",
+    "no-empty-static-block": "warn",
+    "react/no-render-return-value": "warn",
+    "no-sparse-arrays": "warn",
+    "no-constant-binary-expression": "warn",
+    "no-useless-escape": "warn",
+    "use-isnan": "warn",
+    "oxc/number-arg-out-of-range": "warn",
+    "oxc/bad-min-max-func": "warn",
+    "no-this-before-super": "warn",
+    "react/jsx-no-duplicate-props": "warn",
+    "no-extra-boolean-cast": "warn",
+    "no-obj-calls": "warn",
+    "oxc/const-comparisons": "warn",
+    "unicorn/no-useless-length-check": "warn",
+    "no-delete-var": "warn",
+    "react/jsx-no-undef": "warn",
+    "no-global-assign": "warn",
+    "typescript/no-unsafe-declaration-merging": "warn",
+    "no-async-promise-executor": "warn",
+    "no-unsafe-finally": "warn",
+    "no-dupe-else-if": "warn",
+    "typescript/no-duplicate-enum-values": "warn",
+    "no-invalid-regexp": "warn",
+    "unicorn/prefer-string-starts-ends-with": "warn",
+    "no-caller": "warn",
+    "no-self-assign": "warn",
+    "no-compare-neg-zero": "warn",
+    "no-unused-vars": "warn",
+    "no-empty-pattern": "warn",
+    "react/no-is-mounted": "warn",
+    "oxc/bad-array-method-on-arguments": "warn",
+    "unicorn/no-await-in-promise-methods": "warn",
+    "no-shadow-restricted-names": "warn",
+    "no-const-assign": "warn",
+    "unicorn/no-thenable": "warn",
+    "typescript/triple-slash-reference": "warn",
+    "react/no-string-refs": "warn",
+    "no-new-native-nonconstructor": "warn",
+    "oxc/bad-comparison-sequence": "warn",
+    "oxc/uninvoked-array-callback": "warn",
+    "unicorn/no-empty-file": "warn",
+    "unicorn/no-unnecessary-await": "warn",
+    "no-nonoctal-decimal-escape": "warn",
+    "oxc/bad-replace-all-arg": "warn",
+    "unicorn/no-invalid-remove-event-listener": "warn",
+    "no-debugger": "warn",
+    "react/jsx-no-target-blank": "warn",
+    "no-func-assign": "warn",
+    "unicorn/no-useless-fallback-in-spread": "warn",
+    "oxc/erasing-op": "warn",
+    "unicorn/prefer-set-size": "warn",
+    "unicorn/no-useless-spread": "warn",
+    "no-dupe-class-members": "warn",
+    "react/jsx-props-no-spread-multi": "warn",
+    "typescript/no-useless-empty-export": "warn",
+    "react/void-dom-elements-no-children": "warn",
+    "unicorn/no-new-array": "warn",
+    "no-unsafe-optional-chaining": "warn",
+    "no-duplicate-case": "warn",
+    "react/no-danger-with-children": "warn",
+    "typescript/prefer-as-const": "warn",
+    "no-class-assign": "warn",
+    "no-unused-private-class-members": "warn",
+    "typescript/no-extra-non-null-assertion": "warn",
+    "react/no-find-dom-node": "warn",
+    "no-setter-return": "warn",
+    "typescript/no-misused-new": "warn",
+    "unicorn/no-document-cookie": "warn",
+    "oxc/bad-char-at-comparison": "warn",
+    "no-constant-condition": "warn",
+    "no-useless-rename": "warn",
+    "no-ex-assign": "warn",
+    "valid-typeof": "warn"
+  },
```

Comment:
This snapshot will change every time we change a rule, which is not ideal.

Reply:
only when the rules comes from the default plugin and the category `correctness`. 
I think it is not that big of a problem. What do you think?